src/sql/workbench/api/common/extHostModelView.ts

```diff
@@ -1544,7 +1544,7 @@ class DeclarativeTableWrapper extends ComponentWrapper implements azdata.Declara
 		super(proxy, handle, ModelComponentTypes.DeclarativeTable, id, logService);
 		this.properties = {};
 		this._emitterMap.set(ComponentEventType.onDidChange, new Emitter<any>());
-		this._emitterMap.set(ComponentEventType.onDidClick, new Emitter<any>());
+		this._emitterMap.set(ComponentEventType.onSelectedRowChanged, new Emitter<any>());
```

Comment:
Let's update the type here from any - it seems like the API definition is using DeclarativeTableRowSelectedEvent so I'd suggest using that (update the onRowSelected getter too)